modules/howtos/examples/fle.js

```diff
@@ -0,0 +1,167 @@
+"use strict";
+
+const couchbase = require("couchbase");
+const crypto = require("cbfieldcrypt/dist/crypto");
```

Comment:
I had to run `npm install` in the cbfieldcrypt directory in my `node-modules` to be able to run my code. I guess because the library seems to be in `Typescript` we need to transpile to JS somehow?
I wasn't able to use the library otherwise so please let me know how I should import this and I will update it accordingly.

I have tried with `require("cbfieldcrypt/lib/crypto")` but I kept getting this error:
```
Error: Cannot find module 'cbfieldcrypt/lib/crypto'
```
Same for all the other imports from `cbfieldcrypt`

Reply:
Once @brett19 replies with the best practice for that, we will also need to update docs/comment in case someone (or indeed us) later wants to run this example?

(e.g. below I see only `npm i cbfieldcrypt` ?)